### ISSUE TITLE
Reference FilesCollection instance via underlying Mongo.Collection instance

### ISSUE THREAD

**Issue by jankapunkt** [VeliovGroup/Meteor-Files#464]
Note: I would provide a PR but I am not CoffeeScript experienced at all and find it very hard to find the parts of the code to change, in order to implement my feature request.

**Describe your feature / request**

Would it be any problem, to reference the FileCollection instance to the underlying Mongo.Collection instance? 

This would allow users to retrieve the FilesCollection as 'parent' of the Mongo.Collection instance. Therefore, it needs not to be exposed to global namespace and can be integrated in any collection management class or extension, such as dburles:mongo-collection-instances.

Advantages:

- adds no dependencies to any new package
- backwards compatible since there is no other code changed
- would provide a common convention to link the FilesCollection instance via the Mongo.Collection instance, that can be used by other packages, that make use of FilesCollection

**How you going to use it? Give a usage example(s)**

Link the FilesCollection instance to the Mongo.Collection instance:

For now I do this manually, like so:

```javascript
export const Images = FilesCollection({  collectionName: 'Images' });
Images.collection.filesCollection = Images;
```

And then retrieve the FilesCollection as every other collection instance:

```javascript
const Images = Mongo.Collection.get("Images").filesCollection;
```

I use the mechanism of the examples already in my PR to the [Autoform File package](https://github.com/VeliovGroup/meteor-autoform-file), see https://github.com/VeliovGroup/meteor-autoform-file/pull/28

**Comment by dr-dimitru** [comment]
@jankapunkt , thank you a lot for this suggestion.

I'm going to implement this.
Could you add info to the docs please. I'm not familiar with `dburles:mongo-collection-instances`, as you're using it, I believe you are the best to describe how this will be used.

**Comment by jankapunkt** [comment]
> I'm going to implement this.

Wow, cool thing!

> Could you add info to the docs please. 

As a PR with changes in the REAMDE? Should I wait then until your implementation is up or just make the first step here?

**Comment by dr-dimitru** [comment]
> As a PR with changes in the REAMDE? Should I wait then until your implementation is up or just make the first step here?

No need to wait, as we use `dev` branch for development.
Send your PR for docs update to `dev` branch.

**Comment by dr-dimitru** [comment]
@jankapunkt thank you lot for contribution! Published as [`v1.8.2`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.8.2)

Feel free to close.

Please, support this project by:
 - Star on [GitHub](https://github.com/VeliovGroup/Meteor-Files)
 - Star on [Atmosphere](https://atmospherejs.com/ostrio/files)
 - Check [other packages](https://github.com/search?o=desc&q=topic%3Ameteor-package+org%3AVeliovGroup&s=stars&type=Repositories) we have made
